junifer/markers/ets_rss.py

```diff
@@ -0,0 +1,124 @@
+"""Provide class for root sum of squares of edgewise timeseries."""
+
+# Authors: Leonard Sasse <l.sasse@fz-juelich.de>
+#          Nicolás Nieto <n.nieto@fz-juelich.de>
+#          Sami Hamdan <s.hamdan@fz-juelich.de>
+#          Synchon Mandal <s.mandal@fz-juelich.de>
+# License: AGPL
+
+from typing import Dict, List
+
+import numpy as np
+from scipy.stats import zscore
+
+from ..api.decorators import register_marker
+from ..utils import logger
+from .base import BaseMarker
+from .parcel import ParcelAggregation
+
+
+@register_marker
+class RSSETSMarker(BaseMarker):
+    """Class for root sum of squares of edgewise timeseries.
+
+    Parameters
+    ----------
+    atlas : str
+        The name of the atlas.
+    aggregation_method : str, optional
+        The aggregation method (default "mean").
+
+    """
+
+    def __init__(
+            self, atlas: str,
+            aggregation_method: str = "mean",
+            name: str = None,
+    ) -> None:
+        """Initialize the class."""
+        self.atlas = atlas
+        self.aggregation_method = aggregation_method
+        super().__init__(on=["BOLD"], name=name)
+
+    def get_output_kind(self, input: List[str]) -> List[str]:
+        """Get output kind.
+
+        Parameters
+        ----------
+        input : list of str
+            The input to the marker. The list must contain the
+            available Junifer Data dictionary keys.
+
+        Returns
+        -------
+        list of str
+            The updated list of output kinds, as storage possibilities.
+
+        """
+        outputs = []
+        for t_input in input:
+            if t_input in ["BOLD"]:
+                outputs.append("timeseries")
+            else:
+                raise ValueError(f"Unknown input kind for {t_input}")
```

Comment:
This is not needed here.

Reply:
As it is only one input supported, it should only return ["timeseries"]

Reply:
Makes sense.

Reply:
I have removed this.